README.md

```diff
@@ -688,6 +688,50 @@ Example ESLint configuration:
 }
 ```
 
+### [unicorn/template-indent]
+
+**This rule can be used with certain options.**
+
+This rule will automatically fix the indentation of multiline string templates, to keep them in alignment with the code they are found in. A configurable whitelist is used to ensure no whitespace-sensitive strings are edited.
+
+Prettier deals with:
+
+- html
+- css
+- graphql
+- markdown
+
+Using various tags, functions and comments. See: https://github.com/prettier/prettier/tree/main/src/language-js/embed
+
+`template-indent` by default formats some of the same tagged templates, which causes conflicts. See: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/template-indent.md#options
+
+If you like this rule, it can be used just fine with Prettier as long as you add some extra configuration.
+
+Example ESLint configuration:
+
+<!-- prettier-ignore -->
+```js
+{
+  rules: {
+    "unicorn/template-indent": [
+      "error",
+      {
+        tags: [
+          "outdent",
+          "dedent",
+          "gql",
+          "sql",
+          "styled",
+        ],
+        functions: ["dedent", "stripIndent"],
+        selectors: [],
+        comments: ["HTML", "indent"],
```

Comment:
Shouldn’t `gql`, `styled` and `HTML` be removed?

Reply:
Just realized gql is graphql 😁

Apologies! Never used it